cirq/sim/sparse_simulator_test.py

```diff
@@ -147,7 +147,7 @@ def test_run_invert_mask_measure_not_terminal(dtype):
                 np.testing.assert_equal(result.measurements,
                                         {'m': [[1 - b0, b1]] * 3})
                 assert result.repetitions == 3
-        assert mock_sim.call_count == 12
+        assert mock_sim.call_count > 4
```

Comment:
For this and below, would it be worth a line comment to explain what the assertion is testing and/or why ">4" is expected?

Reply:
Added comments